p2p/src/network.rs

```diff
@@ -248,40 +279,39 @@ where
     async fn handle(&mut self, msg: PeerMessage<T>) {
         use PeerMessage::*;
 
+        debug!(?msg);
```

Comment:
```suggestion
        
```
Instead, just instrument the function with `#[log]`, which should produce trace level logs, (which this should be).